src/mocks.h

```diff
@@ -4,7 +4,7 @@
 #include <stdbool.h>
 
 struct stderred {
-  void (*init)();
+  void (*stderred_init)();
```

Comment:
I am pretty sure you can leave this pointer name as it was, it's collision-free on itself. And it's _test code_-only, not in your `stderred.so`